.github/workflows/docs-publish.yml

```diff
@@ -1,5 +1,8 @@
 name: Publish API documentation
 
+permissions:
+  contents: read
```

Comment:
Is this sufficient? This workflow pushes to the gh-pages branch.

Reply:
I think so because we push from a ssh key and not from the token obtained on the running workflow

Reply:
Ok, we will see :)